### ISSUE TITLE
Photo zoom wonky in Google Photos

### ISSUE THREAD

**Issue by capouch** [brave/brave-browser#1281]
## Description 

"Photo zoom" function in Google Photos is erratic and fails intermittently.  After enough tries it will finally zoom.

## Steps to Reproduce

   1. Navigate to a Google Photo image
   2. Use mouse wheel to attempt to zoom
   3. Note erratic behavior

## Actual result:

Hard to describe: sometimes it will work just fine, but mostly you have to zoom-click-zoom-click several times (sometimes *many* times) before the image will respond.  The "Zoom Level Icon" on the upper right will post and disappear intermittently, too.

## Expected result:
Normal zoom functionality.

## Reproduces how often: 
Always

## Brave version (chrome://version info)
0.55.6

### Reproducible on current release:
No.

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
No.
- Is the issue reproducible on the latest version of Chrome? 
No.

### Additional Information
Running Linux Slackware-current (~14.2) as a virtual machine under Virtualbox

**Comment by capouch** [comment]
After playing with it a bit more, the zoomed-in images are blurred, too.  I thought at first it was my camera skills, but when I look at the images with Chrome they look fine.

**Comment by capouch** [comment]
Because I use this feature all the time, I'm able to add a somewhat strange detail that so far appears to be a consistent aspect of this bug: the very first time I navigate to Google Photos after launching the browser, on the very first image I browse, when I invoke the zoom there is a *tiny* hesitation, and then it works fine.  On subsequent image views, though, it is very dodgy, sometimes taking many repeated stabs before it will finally zoom.  

In other words, there seems to be a pattern to it, but it is hard to figure out just what it is.
